docs/embedding/embedded-analytics-js.md

```diff
@@ -31,8 +32,8 @@ Currently you can choose to embed:
 ### 2. Create a new embed
 
 1. In Metabase, click on **+ New** button in top right corner and select **Embed**. Note that this will only be visible to admins.
-2. Choose the type of entity to embed. Currently the available types are dashboard, question, and exploration (which will embed the Metabase query builder).
-3. Next, select the entity you want to embed.
+2. Choose the type of entity to embed. Currently the available types are dashboard, question, and exploration (which will embed the Metabase query builder), and browser.
+3. Next, select the entity you want to embed. For browser, pick the collection you want people to start from.
```

Comment:
Steps 2 and 3 are the same. Should be consolidated, like:

```suggestion
2. Choose the type of entity to embed:
    - dashboard
    - question
    - exploration (which will embed the Metabase query builder)
    - browser (you'll need to pick the collection you want people to start from).
```

What is the default collection for the browser? `root`?

(Also, you may need to format the list.